src/plans/primal_dual_plan.jl

```diff
@@ -1,3 +1,5 @@
+abstract type AbstractPrimalDualProblem{T} <: Problem{T} end
+
 @doc raw"""
     PrimalDualProblem {T, mT <: Manifold, nT <: Manifold} <: PrimalDualProblem} <: Problem{T}
```

Comment:
This line is outdated and should be updated, since you introduces the abstract PD problem.

Reply:
I'm a bit confused here with the comment above. Would you please clarify what should happen? (what is outdated and what needs a docstring)

Reply:
So the thing in the first line (comment above this one) is a new struct that needs a documentation (docstring) and an idea where to introduce that in the actual docs.

but also here in line 4 in the very end it still reads (in the doctoring (` <: Problem{T}`) which is not completely correct if one looks at the (new) actually definition of the problem.

Reply:
AbstractPrimalDualProblem:
- has docs now
- docs are called in problem.md
- Changed the docs of some functions that accept the Abstract problem whereas this was documented in the old way (just primalDualProblem)